README.md

```diff
@@ -14,6 +14,8 @@ The default behaviour is to use the [latest crystal release](https://github.com/
 
 To learn more about using custom buildpacks in Heroku, read [their docs](https://devcenter.heroku.com/articles/third-party-buildpacks#using-a-custom-buildpack).
 
+See also this blog post: http://crystal-lang.org/2016/05/26/heroku-buildpack.html
```

Comment:
the text becomes automatic a link with markdown ( prewview here: https://github.com/muescha/heroku-buildpack-crystal/blob/7082ee0fa81d97692a81130fc72bdac256b01b10/README.md ) 

i prefer the full link, so everybody can see its an "official" blog post from `crystal-lang.org`